ImageFeed/ViewControllers/ImagesList/ImageListView.swift

```diff
@@ -18,4 +18,9 @@ final class ImageListViews {
                                               right: 0)
         return tableView
     }()
+    
+    func setTableViewDelegateAndDataSource(_ viewController: UIViewController) {
+        tableView.delegate = viewController as? UITableViewDelegate
+        tableView.dataSource = viewController as? UITableViewDataSource
+    }
```

Comment:
это не очень хорошо, потому что ничего вообще не гарантирует. Лучше написать так, тогда на уровне компиляции ты не сможешь этот метод использовать, если вьюконтроллер(другая сущность) не будет удовлетворять протоколам.

Кроме того, так ты можешь вообще передать два разных класса, которые будут выполнять разные задачи – в целом, так чаще в продакшене и бывает
```suggestion
    func setTableViewDelegateAndDataSource(delegate: UITableViewDelegate, dataSource: UITableViewDataSource) {
        tableView.delegate = viewController
        tableView.dataSource = viewController
    }
```